lib/unfurl-to-attachment.js

```diff
@@ -1,11 +1,14 @@
 const unfurl = require('unfurl.js')
+const urlParser = require('url-parse')
 
 module.exports = async function unfurlToAttachment (url) {
   const data = await unfurl(url)
+  const parsedUrl = new urlParser(url)
+  const baseUrl = parsedUrl.protocol + '//' + parsedUrl.host
 
   const description = (data.ogp && data.ogp.ogDescription) || (data.other && data.other.description)
   const authorName = data.ogp && data.ogp.ogSiteName
-  const authorIcon = data.other && data.other.icon
+  const authorIcon = baseUrl + (data.other && data.other.icon)
```

Comment:
This will return incorrect for all other results. Try console.log for any other URL, say https://github.com/aryannarora .